apps/flappybird.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: MY-LICENSE
+# Copyright (C) YEAR(S), AUTHOR
+from micropython import const
+import wasp
+import drivers 
+import micropython
+import machine
+import ble
+import time
+print(dir(ble))
+
+@micropython.native
```

Comment:
is `native` really needed? space is precious

Reply:
well originally the game  had better graphics (and needed it for performance ) , after some testing i found that performance stayed almost the same  though (now that it has a simpler design) 'ill remove it